packages/ui-lib/src/Global/atoms/Layout.tsx

```diff
@@ -8,6 +8,7 @@ export const Layout = styled.div`
   display: flex;
   flex-direction: row;
   height: 100%;
+  ${({ theme }) => theme.styles.global.background};
```

Comment:
I didn't consider to fully remove the theme in this PR.
I can do a follow up PR for that replacement since there are some considerations behind that update.

One example is this, that global background value is grey-1.
grey-1 is a variable without context, global theme path in a sense provides context.
SubBackgrounds right now are grey-2 and grey-3.
At the same time other components might have many colors and also include gray 1 without been a background.

Regarding this iteration I wanted to keep just base variables but maybe we could have other sub variables that are global-background, background- 2 or something like that...

I think to figure that is better to do it in a following PR.